One_String_Shifted_to_another_string.rb

```diff
@@ -0,0 +1,26 @@
+=begin
+    
+This problem was asked by Google.
+
+Given two strings A and B, return whether or not A can be shifted some number of times to get B.
+
+For example, if A is abcde and B is cdeab, return true. If A is abc and B is acb, return false.
+    
+=end
+
+
+def string(a, b)
+  array = a.split("")
+  limit = array.length * array.length
+  (limit).times do |x|
+    for x in 0..array.length - 1
+        array[x], array[x - 1] = array[x - 1], array[x] if array.join != b    

```

Comment:
@Anishluke92 can you explain the algorithm in English?

Reply:
1. Take two strings  A and B.
2. In string A, let take change the first character of the string and shift it towards the end of the string. Similar we do for all the characters in the string. 
3. As we shift each characters once we should check whether it matches with the string B.